match/lib/match/encrypt.rb

```diff
@@ -93,7 +93,14 @@ def crypt(path: nil, password: nil, encrypt: true)
       command << "-out #{tmpfile.shellescape}"
       command << "-a"
       command << "-d" unless encrypt
-      command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show an error message if something goes wrong
+      # to show an error message if something goes wrong
+      unless FastlaneCore::Globals.verbose?
+        if !Helper.is_windows?
```

Comment:
nitpick:

```
if windows
  ...
else
  ...
```

Reply:
I did not do that as the code always is "by default assume macOS / Linux" - so I thought it would be better to have that code come first. Disagree?